SwiftUIHomeworks/SwiftUIHomeworks/Custom Views/CustomSegmentedPickerView.swift

```diff
@@ -0,0 +1,84 @@
+// CustomSegmentedPickerView.swift
+// Copyright © RoadMap. All rights reserved.
+
+import SwiftUI
+
+/// Gradient background view
```

Comment:
```suggestion
/// Gradient background view
```
Это кастомные Пикер а по описанию бекграунд с градиентом?